MdePkg/Include/Guid/DebugImageInfoTable.h

```diff
@@ -25,7 +25,7 @@
 #define EFI_DEBUG_IMAGE_INFO_UPDATE_IN_PROGRESS  0x01
 #define EFI_DEBUG_IMAGE_INFO_TABLE_MODIFIED      0x02
 
-#define EFI_DEBUG_IMAGE_INFO_TYPE_NORMAL  0x01
+#define EFI_DEBUG_IMAGE_INFO_TYPE_NORMAL  0x02

```

Comment:
Maybe just add a new definition with “V2”? Or maybe call the old one LEGACY_PE?

Reply:
Renamed the old one in new force pushed commit.